odoo/addons/base/wizard/base_module_uninstall_views.xml

```diff
@@ -16,27 +16,24 @@
                         <div class="me-auto p-2 bd-highlight"><h3>Apps to Uninstall</h3></div>
                         <div class="p-2 bd-highlight"><field name="show_all"/> Show All</div>
                     </div>
-                    <field name="module_ids" mode="kanban" class="o_modules_field">
+                    <field name="module_ids" mode="kanban">
                         <kanban create="false" class="o_modules_kanban">
-                            <field name="icon"/>
                             <field name="state"/>
-                            <field name="summary"/>
                                 <templates>
-                                    <t t-name="kanban-box">
-                                        <div class="oe_module_vignette">
-                                            <t t-set="installed" t-value="record.state.raw_value == 'installed'" />
-                                            <img t-attf-src="#{record.icon.value}" class="oe_module_icon" alt="Icon" />
-                                            <div class="oe_module_desc" t-att-title="record.shortdesc.value">
-                                                <h4 class="o_kanban_record_title">
-                                                    <field name="shortdesc" />&amp;nbsp;
-                                                </h4>
-                                                <p class="oe_module_name">
-                                                    <field groups="!base.group_no_one" name="summary" />
-                                                    <code groups="base.group_no_one">
-                                                        <field name="name" /></code>
-                                                </p>
-                                            </div>
-                                        </div>
+                                    <t t-name="kanban-card" class="flex-row align-items-center">
+                                        <t t-set="installed" t-value="record.state.raw_value == 'installed'" />
```

Comment:
installed is not used?

Reply:
No,installed is not used.i can't find any usecase for it. should i remove it?

Reply:
yes please.